### DISCUSSION TITLE
Icons

### DISCUSSION THREAD

**Discussion by disimmon** [microsoft/atlas-design#401] [Component]
We primarily use MDL2 icons on the site (current incomplete list for reference https://docsstyleguide.z13.web.core.windows.net/icon-font.html), but many of the new designs prefer Fluent icons (https://iconcloud.design/search/flag). Should there be a conversion of MDL2 icons to Fluent icons, or is a mix / MDL2 preferred? IMO the fluent icons look a bit more modern than the MDL2 counters but perhaps there is a place or convention for both.

**Comment by valerietrutherford** [comment]
As far as I know, Fluent MDL2 set isn't being maintained and was meant as a stop-gap for products to have more time to build the SVG sets into their product, and I don't think there are plans to add to MDL2. Fluent Iconography team tells us to use Fluent icons (https://github.com/microsoft/fluentui-system-icons) for products that use Fluent components. So conversion to Fluent icons would make the most sense if our components want to align with Fluent's.

**Reply by ValerieRutherford** [reply]
> `test code background color`

> test quote background color

**Reply by ValerieRutherford** [reply]
Another test

**Reply by ValerieRutherford** [reply]
Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Orci a scelerisque purus semper eget. Fermentum odio eu feugiat pretium nibh ipsum consequat nisl. Felis donec et odio pellentesque. Netus et malesuada fames ac turpis egestas sed tempus urna. Arcu vitae elementum curabitur vitae nunc sed velit dignissim. Ultricies leo integer malesuada nunc vel risus commodo viverra. Volutpat consequat mauris nunc congue nisi vitae suscipit tellus mauris. Rutrum

**Reply by ValerieRutherford** [reply]
Azure tenant is a directory, subscription is an object that represents a "folder" that you can put resources in. Subscriptions are tied to tenants. so 1 tenant can have many subscriptions, but not vice versa. Link: https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits

**Reply by ValerieRutherford** [reply]
In this article, you use Visual Studio Code to create a Python function that responds to HTTP requests. After testing the code locally, you deploy it to the serverless environment of Azure Functions.

This article covers both Python programming models supported by Azure Functions. Use the selector at the top to choose your programming model.

**Reply by ValerieRutherford** [reply]
Before you begin, make sure that you have the following requirements in place:

An Azure account with an active subscription. [Create an account for free](https://azure.microsoft.com/free/?ref=microsoft.com&utm_source=microsoft.com&utm_medium=docs&utm_campaign=visualstudio).
The [Azure Functions Core Tools](https://learn.microsoft.com/en-us/azure/azure-functions/functions-run-local#install-the-azure-functions-core-tools) version 4.x.
Python versions that are [supported by Azure Functions](https://learn.microsoft.com/en-us/azure/azure-functions/supported-languages#languages-by-runtime-version). For more information, see [How to install Python](https://wiki.python.org/moin/BeginnersGuide/Download).

[Visual Studio Code](https://code.visualstudio.com/) on one of the [supported platforms](https://code.visualstudio.com/docs/supporting/requirements#_platforms).

The [Python extension](https://marketplace.visualstudio.com/items?itemName=ms-python.python) for Visual Studio Code.

The [Azure Functions extension](https://marketplace.visualstudio.com/items?itemName=ms-azuretools.vscode-azurefunctions) for Visual Studio Code, version 1.8.3 or a later version.

**Comment by disimmon** [comment]
It would be good if we could get a 1:1 mapping of the icons we have now in docons vs the new fluent icons. The allowlist of icons are handled in a seperate library that is old and a pain to update so we would want to do a batch update if possible. This is a nearly complete list of current icons: https://docsstyleguide.z13.web.core.windows.net/icon-font.html. There are probably 10-15 new icons not covered on that page that were added after we stopped maintaining the legacy styleguide.
